collectors/gsuite/README.md

```diff
@@ -10,35 +10,57 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. G Suite Setup
+
+1. Create a Custom admin role. [Link](https://support.google.com/a/answer/1219251?hl=en&ref_topic=4514341)
```

Comment:
I'd suggest replacing the link here with this https://support.google.com/a/answer/2406043